### PR TITLE
Properly configure mock for el7.3

### PR DESCRIPTION
**Pull request by nichenke**
This is a set of tweaks to get a working el7.3 mock config.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,4 @@
 # docker-rpm-build
 Docker container to help build Versity RPMS in mock
+
+Should checkout https://github.com/mmornati/docker-mock-rpmbuilder for more ideas.
```
Comment (heemstra) [inline README.md]: Hate to say this when I have code out for review myself, but... shouldn't this be a separate commit?  I followed the link hoping it said something about /usr/bin vs /usr/sbin but, alas, no.

Comment (nichenke) [reply README.md]: Gack, yes. Will fix.
